shared/common/build/tools/generate_headers.sh

```diff
@@ -127,6 +127,9 @@ for file in $(find $wrapper_directory -type f); do
     if [[ $filename == $wrapper_filename ]]; then
         continue;
     fi
+    if [[ $filename == "$OPT_NAME-Swift.h" ]]; then
```

Comment:
Swift header shouldn’t be included in the umbrella header, the swift header imports the umbrella header

Reply:
Circular dependency avoidance! Could you add a comment here, explaining that?